lms/envs/common.py

```diff
@@ -392,6 +392,9 @@
     # Whether to check the "Notify users by email" checkbox in the batch enrollment form
     # in the instructor dashboard.
     'BATCH_ENROLLMENT_NOTIFY_USERS_DEFAULT': True,
+
+    # Whether the bulk enrollment view is enabled.
+    'ENABLE_BULK_ENROLLMENT_VIEW': False,
```

Comment:
Since feature flags are pulled in from the configuration as the FEATURES dict, I don't think they usually need anything added besides a default in `common.py` unless they trigger some additional config action, like including a djangoapp. I can open a pull request to the configuration repo and plumb it through there once this is merged, though.